scripts/update-config-page.mjs

```diff
@@ -0,0 +1,315 @@
+
+<Tabs groupId={"configuration"}>
+<TabItem value={"configuration file"} label={"Configuration File"}>
+
+You can configure the OpenFGA server with a \`config.yaml\` file, which can be specified in either:
+- \`/etc/openfga\`
+- \`$HOME/.openfga\`
+- \`.\` (i.e., the current working directory).
+
+The OpenFGA server will search for the configuration file in the above order.
+
+Here is a sample configuration to run OpenFGA with a Postgres database and using a preshared key for authentication:
+
+\`\`\`yaml
+datastore:
+  engine: postgres
+  uri: postgres://user:password@localhost:5432/mydatabase
+authn:
+  method: preshared
+  preshared:
+    keys: ["key1", "key2"]
+playground:
+  enabled: false
+\`\`\`
+
+</TabItem>
+
+<TabItem value={"environment variables"} label={"Environment Variables"}>
+
+The OpenFGA server supports **environment variables** for configuration, and they will take priority over your configuration file.
+Each variable must be prefixed with \`OPENFGA_\` and followed by your option in uppercase (\`datastore.engine\` becomes \`OPENFGA_DATASTORE_ENGINE\`), e.g.
+
+\`\`\`shell
+# Running as a binary
+export OPENFGA_DATASTORE_ENGINE=postgres
+export OPENFGA_DATASTORE_URI='postgres://postgres:password@postgres:5432/postgres?sslmode=disable'
+export OPENFGA_AUTHN_METHOD=preshared
+export OPENFGA_AUTHN_PRESHARED_KEYS='key1,key2'
+export OPENFGA_PLAYGROUND_ENABLED=false
+openfga run
+
+# Running in docker
+docker run docker.io/openfga/openfga:latest \\\ 
+  -e OPENFGA_DATASTORE_ENGINE=postgres \\\ 
+  -e OPENFGA_DATASTORE_URI='postgres://postgres:password@postgres:5432/postgres?sslmode=disable' \\\ 
+  -e OPENFGA_AUTHN_METHOD=preshared \\\ 
+  -e OPENFGA_AUTHN_PRESHARED_KEYS='key1,key2' \\\ 
+  -e OPENFGA_PLAYGROUND_ENABLED=false \\\ 
+  run
+\`\`\`
+
+</TabItem>
+
+<TabItem value={"command line parameters"} label={"Command Line Parameters (Flags)"}>
+
+Command line parameters take precedence over environment variables and options in the configuration file. They are prefixed with \`--\` (\`OPENFGA_DATASTORE_ENGINE\` becomes \`--datastore-engine\`), e.g.
+
+\`\`\`shell
+# Running as a binary
+openfga run \\\ 
+  --datastore-engine postgres \\\ 
+  --datastore-uri 'postgres://postgres:password@postgres:5432/postgres?sslmode=disable' \\\ 
+  --authn-method=preshared \\\ 
+  --authn-preshared-keys='key1,key2' \\\ 
+  --playground-enabled=false
+
+# Running in docker
+docker run docker.io/openfga/openfga:latest run \\\ 
+  --datastore-engine postgres \\\ 
+  --datastore-uri 'postgres://postgres:password@postgres:5432/postgres?sslmode=disable' \\\ 
+  --authn-method=preshared \\\ 
+  --authn-preshared-keys='key1,key2' \\\ 
+  --playground-enabled=false
+\`\`\`
+
+</TabItem>
+</Tabs>
+
+`;
+        let mdxContent = mdxContentHeader + `## List of options
+
+The following table lists the configuration options for the OpenFGA server, based on the [config-schema.json](https://github.com/openfga/openfga/blob/main/.config-schema.json).
```

Comment:
What do you think to having this point to the tag we generated from if we passed in the `release` value we get?